server/tests/models/article.spec.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable no-console */
+import chai from 'chai';
+import chaiAsPromise from 'chai-as-promised';
+import models from '../../models';
+import dummyData from '../../fixtures/models/article';
+
+chai.use(chaiAsPromise);
+const { expect } = chai;
+const { Article, sequelize } = models;
+const { articleData } = dummyData;
+beforeEach(async () => {
+  try {
+    await sequelize.sync({ force: true });
```

Comment:
maybe you can research a setup Test or a helper so this action is not repeated written in multiple tests